src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -98,12 +98,24 @@ protected function get_properties() {
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
+			'remaining_stock'     => [
+				'description' => __( 'Quantity left in stock or null if not applicable.', 'woo-gutenberg-products-block' ),
```

Comment:
Nit: `, or`? like the below description.

Reply:
Does this make `low_stock_remaining` redundant?

More importantly, does it matter than we're revealing stock amounts via our API? Could this be sensitive? I think this was some of the reasoning behind only returning low_stock_remaining - there is a threshold where this data is not visible.

Reply:
🤔 ya as per discussion in slack, (p1583769076089600-slack-C8X6Q7XQU) we'll remove the remaining stock exposure for now and stick to using `low_stock_remaining` to limit quantities if that's available.